ion/symboltoken.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ion
+
+var UnknownSid int64 = -1
```

Comment:
Even better: `const UnknownSID = -1`. :)

(Actually, could you use 0 here instead of -1? I think so, and that'd play nicely in that `SymbolToken{}` would be the right way to initialize a default, invalid symbol token.)

Reply:
Updated to const.

We have to use unknown SID as -1 because 0 is for symbol zero which is used to denote that a SymbolToken has unknown text in any symbol table.